core/tests/protractor_utils/CollectionEditorPage.js

```diff
@@ -135,32 +140,40 @@ var CollectionEditorPage = function() {
   this.publishCollection = async function() {
     await waitFor.elementToBeClickable(
       editorPublishButton, 'Collection Publish button is not clickable');
-    await editorPublishButton.click();
+    await action.click('Editor Publish Button', editorPublishButton);
   };
 
   // Set collection title.
   this.setTitle = async function(title) {
-    await editorTitleInput.sendKeys(title);
+    await action.sendKeys('Editor Title Input', editorTitleInput, title);
   };
 
   // Set collection objective.
   this.setObjective = async function(objective) {
-    await collectionEditorObjectiveInput.sendKeys(objective);
+    await action.sendKeys(
+      'Collection Editor Objective Input',
+      collectionEditorObjectiveInput, objective);
   };
 
   // Set collection category.
   this.setCategory = async function(category) {
-    await editorCategoryDropdown.first().click();
+    await action.click(
+      'Editor Category Drop Down', editorCategoryDropdown.first());
+    // Below line needs to be corrected under this issue:
+    // https://github.com/oppia/oppia/issues/10798 but it throws Error
+    // "Failed: Cannot read property 'bind' of undefined"on this change:-
+    // await action.sendKeys('Category Item',
+    // await browser.driver.switchTo().activeElement(),category + '\n');
```

Comment:
I would suggest re-wording the comment a bit:
```suggestion
   // The following sendKeys statement cannot be converted to action.sendKeys(..) 
   // because doing so causes the e2e test to fail with the error
   // "Failed: Cannot read property 'bind' of undefined".
```

Reply:
Also, could you please create an issue to figure out a way to fix the error mentioned (`"cannot read property 'bind' ..`) and assign it to me please? In addition to the re-wording suggestion above, could you also link the newly created issue as a TODO (see examples in the codebase for this). Thanks!